.github/workflows/build-boost-packages.yml

```diff
@@ -125,8 +125,8 @@ jobs:
         $boostDirectory = GetToolDirectory -ToolName "boost" -Version "${{ github.event.inputs.VERSION }}" -Architecture "${{ matrix.architecture }}"
         $LD_LIBRARY = Join-Path -Path $boostDirectory -ChildPath "lib"
         Write-Host "BOOST_ROOT = ${boostDirectory}"
-        echo "::set-env name=BOOST_ROOT::${boostDirectory}"
-        echo "::set-env name=LD_LIBRARY_PATH::${LD_LIBRARY}"
+        echo "BOOST_ROOT=${boostDirectory}" >> $GITHUB_ENV
```

Comment:
@thboop , Are you sure that `$GITHUB_ENV` should be available?
Should it be `$env: GITHUB_ENV` since it is pwsh?